### DISCUSSION TITLE
[suggestion] Add some images to the readme or the website.

### DISCUSSION THREAD

**Discussion by K0-RR** [pachli/pachli-android#488] [Ideas]
Hello, I noticed there are great screenshots in https://github.com/pachli/pachli-android/tree/main/fastlane/metadata/android/en-US/images/phoneScreenshots but probably only developers would be able to find them.

**Comment by nikclayton** [comment]
Hi, thanks for the suggestion (and the compliment about the screenshots)

Those screenshots show up on the Google Play and F-Droid pages for the app, so they will appear when users are looking for them.

What do you think of https://gist.github.com/nikclayton/d2fae97d55bcb116967d4efa0d13bacf as an alternative?

**Comment by nikclayton** [comment]
Closing this, since the README was updated to include the images.
